application/src/test/java/org/opentripplanner/updater/trip/RealtimeTestEnvironmentBuilder.java

```diff
@@ -24,32 +28,31 @@
 import org.opentripplanner.transit.model.timetable.TripTimes;
 import org.opentripplanner.transit.model.timetable.TripTimesFactory;
 import org.opentripplanner.transit.service.TimetableRepository;
+import org.opentripplanner.updater.trip.FlexTripInput.FlexStop;
 
 public class RealtimeTestEnvironmentBuilder {
```

Comment:
Same concern again, why do we use a realtime test environment for flex stuff?

Reply:
The RealTimeTestEnvironement is the easiest way to build a full `TransitService` so you don't have to build the whole data model (stop times, trip, route, agency, trip times, timetables...) by hand.

The other option is to make a builder just for flex stuff but that would share much of the same code. What do you think of elevating this to a general purpose "data model builder" and renaming it?

Reply:
We discussed this on Tuesday and it's fine to use this test environment here for now.